### PR TITLE
Change output of errors for native ssh (#1012)

### PR DESCRIPTION
**Pull request by pluseg**
| Q             | A
| ------------- | ---
| Bug fix?      | No
| New feature?  | Yes
| BC breaks?    | No
| Deprecations? | No
| Fixed tickets | #1012

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -18,8 +20,6 @@
 - Fixed possibility to use PEM files with Native SSH
 - Fixed old releases not being cleaned up when keep_releases reduced by more than half.
 
-### Changed
-- Add task queue:restart for Laravel recipe [#1007](https://github.com/deployphp/deployer/pull/1007)
```
Comment (pluseg) [inline CHANGELOG.md]: Just combined with same section above.

File: src/Server/Remote/NativeSsh.php [hunk 2]
```diff
@@ -49,7 +50,6 @@ public function run($command)
         $serverConfig = $this->getConfiguration();
         $sshOptions = [
             '-A',
-            '-q',
```
Comment (pluseg) [inline src/Server/Remote/NativeSsh.php]: @elfet Is there a real reason of adding "-q" option? Without it `ssh` returns 255 status code (Unknown error) and empty output.

Comment (elfet) [inline src/Server/Remote/NativeSsh.php]: Don't know 🙄 It came from PR.

File: src/Server/Remote/NativeSsh.php [hunk 3]
```diff
@@ -85,12 +85,17 @@ public function run($command)
...
-            ->mustRun();
+        try {
+            $process = new Process($sshCommand);
+            $process
+                ->setPty($serverConfig->getPty())
+                ->setTimeout(null)
+                ->setIdleTimeout(null)
+                ->mustRun();
+        } catch (ProcessFailedException $exception) {
+            $errorMessage = \Deployer\isVerbose() ? $exception->getMessage() : $process->getErrorOutput();
```
Comment (elfet) [inline src/Server/Remote/NativeSsh.php]: Why in verbose mode using message and otherwise error output?

Comment (pluseg) [inline src/Server/Remote/NativeSsh.php]: @elfet Error output returns only error string like "invalid credentials" and an exception message has current look with full broken command, current directory, error string, etc. I think that common user needs only concise error string.

Comment (elfet) [inline src/Server/Remote/NativeSsh.php]: But why not isDebug then?

Comment (pluseg) [inline src/Server/Remote/NativeSsh.php]: @elfet I used it but then noted that there is one usage of Verbose in the file. Okay, I'll update the PR.

**Pull request merged**